packages/venia-ui/lib/components/MiniCart/miniCart.gql.js

```diff
@@ -34,10 +30,14 @@ export const REMOVE_ITEM_MUTATION = gql`
             cart {
                 id
                 ...MiniCartFragment
+                ...CartPageFragment
+                ...CheckoutPageFragment
```

Comment:
I imagine that checkout page won't actually update correctly because this fragment is _just_ `total_quantity`. Does shipping methods and the value change when you remove an item? What happens if you have a gift card applied that does not cover the value and then remove an item that makes the gift card cover the entire value?

Reply:
Mini Cart will not be available on the checkout page. That should solve the problem but if someone in the future wants mini cart on the checkout page they should make sure to request required data in the mutation.

Reply:
Is there a ticket for disabling it on the checkout page? 

Also, if it won't be available on the checkout page then remove the fragment.

Reply:
Here it is: https://jira.corp.magento.com/browse/PWA-737

I wanted to remove it while working on that ticket. Till then let's have it.